frontend/components/groups/css/groups.css

```diff
@@ -136,8 +136,8 @@ groups .group:hover {
 
 groups .group {
     border-bottom: solid gainsboro 1px;
-    min-height: 95px;
-    max-height: 95px;
+    min-height: 120px;
+    max-height: 120px;
```

Comment:
25 px is probably too excessive, i think you need like another 5px maybe to make the desc visible?

I think it's better if we can reduce the spacing between the group name, author and description, as that's a lot of padding between them.